uber/common.py

```diff
@@ -16,7 +17,7 @@
 import traceback
 from glob import glob
 from uuid import uuid4
-from io import StringIO
+from io import StringIO, BytesIO
```

Comment:
Can you move this down to be consistent with our ordering?  These imports are all in increasing order of line length.  I find it very soothing to look at.  (That sounds like a joke, but I genuinely like how it looks to see the import lines getting monatomically longer.